src/pages/loading-waiting-states/skeletons/Skeletons.tsx

```diff
@@ -0,0 +1,359 @@
+        <>
+            <AppBar data-cy="pxb-toolbar" position={'sticky'} classes={{ root: classes.appbarRoot }}>
+                <Toolbar classes={{ gutters: classes.toolbarGutters }}>
+                    <Hidden mdUp={true}>
+                        <IconButton
+                            data-cy="toolbar-menu"
+                            color={'inherit'}
+                            onClick={(): void => {
+                                dispatch({ type: TOGGLE_DRAWER, payload: true });
+                            }}
+                            edge={'start'}
+                            style={{ marginRight: 20 }}
+                        >
+                            <Menu />
+                        </IconButton>
+                    </Hidden>
+                    <Typography variant={'h6'} color={'inherit'}>
+                        Skeletons
+                    </Typography>
+                    <Spacer />
+                    <Tooltip title={'Refresh'}>
+                        <span>
+                            <IconButton edge={'end'} color={'inherit'} onClick={refreshData} disabled={isLoading}>
+                                <Refresh />
+                            </IconButton>
+                        </span>
+                    </Tooltip>
+                </Toolbar>
+            </AppBar>
+            <div className={classes.exampleContainer}>
+                <div className={classes.buttonRow}>
+                    <Typography variant={'subtitle1'} className={classes.label}>
+                        Animation Style
+                    </Typography>
+                    <ButtonGroup color="primary">
+                        <Button
+                            onClick={(): void => setAnimationStyle('pulse')}
+                            className={animationStyle === 'pulse' ? classes.selected : ''}
+                        >
+                            Pulse
+                        </Button>
+                        <Button
+                            onClick={(): void => setAnimationStyle('wave')}
+                            className={animationStyle === 'wave' ? classes.selected : ''}
+                        >
+                            Wave
+                        </Button>
+                    </ButtonGroup>
+                </div>
+                <Typography variant={'h6'} className={classes.title}>
+                    Scorecard
+                </Typography>
+                {isLoading && (
+                    <div>
+                        <ScorecardPlaceholder animation={animationStyle} />
+                    </div>
+                )}
+                {!isLoading && (
+                    <ScoreCard
+                        style={{ width: 400, flex: '0 0 auto' }}
+                        headerTitle={'Substation 3'}
+                        headerSubtitle={'High Humidity Alarm'}
+                        headerInfo={'4 Devices'}
+                        headerColor={colors.blue[500]}
```

Comment:
I think blue is the default color

Reply:
true story... I removed this definition so it just uses the default.